pmm/pmm2-server-release.groovy

```diff
@@ -185,13 +185,23 @@ pipeline {
                         docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
                         docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${DOCKER_MID}
                         docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${TOP_VER}
```

Comment:
my opinion - we should release into percona only

Reply:
Why?

Reply:
Ok, that's fine